internal/cli/server/config.go

```diff
@@ -131,6 +131,10 @@ type P2PConfig struct {
 
 	// Discovery has the p2p discovery related settings
 	Discovery *P2PDiscovery `hcl:"discovery,block" toml:"discovery,block"`
+
+	// TxArrivalWait sets the maximum duration the transaction fetcher will wait for
+	// an announced transaction to arrive before explicitly requesting it
+	TxArrivalWait time.Duration `hcl:"txarrivalwait,optional" toml:"txarrivalwait,optional"`
```

Comment:
could you check how it works with and without provided by user value, especially `optional` tag?

Reply:
added a log line to print the `txArrivalWait` value inside of the start of the fetcher's go routine

[b8be0da](https://github.com/thogard785/bor/commit/b8be0dace866c6ca1995eb2f344c2beaf0790d5c)

can confirm that no flag being used results in a value of 500ms being used by fetcher. 

can confirm that a nonsensical value for the flag results in bor not launching
`invalid value "purple" for flag -txarrivalwait: parse error` (then prints the acceptable commands)